From 49383bd1ada3195f14c3091526867bc5637453b1 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Mon, 3 Jun 2019 09:16:22 -0700
Subject: [PATCH 5/5] DO-NOT-UPSTREAM: scsi: qla2xxx: Hide a
 -Wframe-larger-than= warning

I'll try to remember to file an issue and look into this then.

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/scsi/qla2xxx/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/scsi/qla2xxx/Makefile b/drivers/scsi/qla2xxx/Makefile
index 17d5bc1cc56b..71f4757f593d 100644
--- a/drivers/scsi/qla2xxx/Makefile
+++ b/drivers/scsi/qla2xxx/Makefile
@@ -5,3 +5,7 @@ qla2xxx-y := qla_os.o qla_init.o qla_mbx.o qla_iocb.o qla_isr.o qla_gs.o \
 
 obj-$(CONFIG_SCSI_QLA_FC) += qla2xxx.o
 obj-$(CONFIG_TCM_QLA2XXX) += tcm_qla2xxx.o
+
+ifdef CONFIG_CC_IS_CLANG
+CFLAGS_qla_nx2.o += -Wframe-larger-than=4096
+endif
-- 
2.22.0

